go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -1014,6 +1016,7 @@ var defaultConfig = TabletConfig{
 	Consolidator:                Enable,
 	ConsolidatorStreamTotalSize: 128 * 1024 * 1024,
 	ConsolidatorStreamQuerySize: 2 * 1024 * 1024,
+	ConsolidatorMaxQueryWait:    256,
```

Comment:
We have to keep the default 0 that should point to unlimited wait. Otherwise it will cause regression with system who depends on high number and during upgrade they would not be able to set this as it is a new configuration.

Reply:
Thanks @harshit-gangal,  it's a good idea, I will going to update that.